script/Arcueid/Day 13/13_00_ARC13_5.txt

```diff
@@ -603,6 +607,7 @@ If you could truly see death―――you wouldn't even be able to stand."
 [sha:0dcfa1045baef15addb3b33f3b7b1ba1aa189726]{
 -- Page 438, Offset 32929.
 -- 喩えるのなら、それは月世界。
+// group pr check: change the title from The Lunar World to The Moon's Surface 
```

Comment:
this was a suggestion so that the literal title of the chapter could be worked in here.
The line would change to "You could liken it to the moon's surface".

Reply:
I feel like surface of the moon is more natural in this phrasing, is that too long to be a chp title?

Reply:
I think The Moon's Surface is a better chapter title and I'm favoring that even though it makes the line read slightly more awkwardly.